.github/ISSUE_TEMPLATE/custom-issue-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Should we fill in these fields? It seems when I click `New Issue`, these lines will show up first:
<img width="2233" alt="image" src="https://github.com/skypilot-org/skypilot/assets/6753189/89874f54-20c8-4ef6-9e45-eea9a849c6f9">

Reply:
Good catch. Now made the template appear directly after clicking New Issue (just like our PR template):
<img width="691" alt="Screen Shot 2024-03-31 at 18 31 18" src="https://github.com/skypilot-org/skypilot/assets/592670/e693a46f-a647-449e-9f14-dc0c59445a3c">